src/test/chain-simulator/events.cs-e2e.ts

```diff
@@ -0,0 +1,107 @@
+import { config } from "./config/env.config";
+import { ChainSimulatorUtils } from "./utils/test.utils";
+import axios from "axios";
+
+describe('Events e2e tests with chain simulator', () => {
+  beforeAll(async () => {
+    await ChainSimulatorUtils.waitForEpoch(2);
+  });
+
+  describe('GET /events', () => {
+    it('should return status code 200 and a list of events', async () => {
+      const response = await axios.get(`${config.apiServiceUrl}/events`);
+      expect(response.status).toBe(200);
+      expect(response.data).toBeInstanceOf(Array);
+    });
+
+    it('should support pagination', async () => {
+      const response = await axios.get(`${config.apiServiceUrl}/events?from=0&size=2`);
```

Comment:
we can add also a check that ?from=0&size=2 & ?from=1&size=2 returns different results

Reply:
done